### DISCUSSION TITLE
command line utility

### DISCUSSION THREAD

**Discussion by chandansql** [upscayl/upscayl#555] [Q&A] [answered]
I would like to automate upscayl in my tool with command line , i want to know know method to call command line with input and output parameter with image path so that i can call this tool through my application , is there any document or artical on this ?

**Comment by aaronliu0130** [comment, marked as answer]
Hi, as you can see in our FAQ:

> - **Is there a CLI available?**
>   - The CLI tool is called real-esrgan-ncnn-vulkan and it's available on the Real-ESRGAN repository.
>   - As of v2.9, we're using our fork of Real-ESRGAN-ncnn-vulkan at https://github.com/upscayl/upscayl-ncnn.
